src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -76,6 +78,101 @@ def chunk_iter(inputs, feature_extractor, chunk_len, stride_left, stride_right):
             yield {"is_last": is_last, "stride": stride, **processed}
 
 
+def _find_timestamp_sequence(sequences, tokenizer, feature_extractor):
+    """ """
+    max_source_positions = 1500
+    begin_idx = 3
+    last_begin_time = 0
+    items = []
+    offsets = []
+    # index of the first timestamp token
+    timestamp_begin = tokenizer.all_special_ids[-1] + 1
+    time_precision = feature_extractor.chunk_length / max_source_positions
+    last_timestamp_position = 0
+    for seq_idx, sequence in enumerate(sequences):
+        timestamp_offset = (seq_idx * feature_extractor.chunk_length) / 2  # in seconds
```

Comment:
I don't think this is a property of the `feature_extractor` the sequences length should depend on pipeline parameters instead (and probably on the chunk_length too, no ?)